test/workbox-build/node/entry-points/generate-sw-string.js

```diff
@@ -436,10 +435,10 @@ describe(`[workbox-build] entry-points/generate-sw-string.js (End to End)`, func
       await validateServiceWorkerRuntime({swCode, expectedMethodCalls: {
         [STRING_HANDLER]: [[{
           cacheName: firstRuntimeCachingOptions.cacheName,
-          plugins: ['workbox.expiration.Plugin'],
+          plugins: [{}],
```

Comment:
FYI regarding this change: we previously had a hardcoded string returned by the stubbed plugin methods in the mock environment used to make this sort of test easier to write.

Now that we're using actual classes instead of stubbed methods (to ensure that `new` is being called), the return value looks like a generic object. We get the same test coverage by virtue of testing the `cacheableResponsePlugin` and `cacheExpirationPlugin` spies on the following lines, though.